src/mediatype-icon.ts

```diff
@@ -1,19 +1,8 @@
 import { css, CSSResultGroup, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
 
-import { accountIcon } from './assets/img/icons/mediatype/account';
-import { audioIcon } from './assets/img/icons/mediatype/audio';
-import { collectionIcon } from './assets/img/icons/mediatype/collection';
-import { dataIcon } from './assets/img/icons/mediatype/data';
-import { etreeIcon } from './assets/img/icons/mediatype/etree';
-import { imagesIcon } from './assets/img/icons/mediatype/images';
-import { filmIcon } from './assets/img/icons/mediatype/film';
-import { radioIcon } from './assets/img/icons/mediatype/radio';
-import { softwareIcon } from './assets/img/icons/mediatype/software';
-import { textsIcon } from './assets/img/icons/mediatype/texts';
-import { tvIcon } from './assets/img/icons/mediatype/tv';
-import { videoIcon } from './assets/img/icons/mediatype/video';
-import { webIcon } from './assets/img/icons/mediatype/web';
+import { mediatypeData } from './mediatype/mediatype-display';
```

Comment:
Blocker: By Lit convention, the class name should match the file name.
So, asked around about a good name, suggestion: 

`import { mediatypeConfig } from './mediatype/mediatype-config';`

Reply:
Please rename either file or name of class to match Lit convention: 
camel case class name => lowercase, -[dash] separated file name.
 so 
 mediatypeData => mediatype-data
 But suggest name change of both: 
 mediatypeConfig ...